master/reference/kube-controllers/configuration.md

```diff
@@ -50,6 +50,7 @@ The following environment variables can be used to configure the {{site.prodname
 | `ENABLED_CONTROLLERS` | Which controllers to run | namespace, node, policy, serviceaccount, workloadendpoint |
 | `LOG_LEVEL`     | Minimum log level to be displayed. | debug, info, warning, error |
 | `KUBECONFIG`    | Path to a kubeconfig file for Kubernetes API access | path |
+| `SYNC_NODE_LABELS`    | Toggles the synchronization of labels between Kubernetes Nodes and Calico Nodes. | boolean |
```

Comment:
"When enabled, Kubernetes node labels will be copied to Calico node objects"

Reply:
To me, the current wording implies that labels will be copied in both directions (arguably a bad config option name...)